source/_components/fan.zha.markdown

```diff
@@ -0,0 +1,15 @@
+---
+layout: page
+title: "ZigBee Home Automation Fan"
+description: "Instructions how to setup ZigBee Home Automation fans within Home Assistant."
+date: 2018-02-10 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: zigbee.png
+ha_category: Fan
+ha_iot_class: "Local Polling"
+---
```

Comment:
`ha_release` missing?

Reply:
Not sure what version I should put here...the platform hasn't been merged yet and I'm not sure if the next release will be a minor or a patch release

Reply:
@igorbernstein2 This a new platform/component, those are never released in a minor release.
So please put in the upcoming next major release (`0.66`).

Reply:
done